examples/files.json

```diff
@@ -121,7 +121,6 @@
 		"webgl_loader_texture_hdr",
 		"webgl_loader_texture_ktx",
 		"webgl_loader_texture_ktx2",
-		"webgl_loader_texture_logluv",
```

Comment:
The respective E2E screenshot also needs to be removed, otherwise the CI complains.

Reply:
Done!